dev/io.openliberty.org.apache.commons.logging_fat/bnd.bnd

```diff
@@ -0,0 +1,37 @@
+#*******************************************************************************
+# Copyright (c) 2022, 2024 IBM Corporation and others.
```

Comment:
That one is borderline since I copied it from another bucket to begin with.  So technically it is a derivative work from another file that is copyright 2022.